cloudpickle/cloudpickle_fast.py

```diff
@@ -328,7 +328,7 @@ def _file_reduce(obj):
         obj.seek(0)
         contents = obj.read()
         obj.seek(curloc)
-    except IOError as e:
+    except OSError as e:
```

Comment:
Since this case is not covered by any test, I would rather not make this change to avoid introducing an unintended change of behavior if not necessary.

Reply:
I'm happy to revert this change to avoid any unintended changes. Though I did want to point out that since Python 3.3 `IOError` (along with a few other built in exceptions) were merged into [`OSError`](https://docs.python.org/3/library/exceptions.html#OSError):

```python
In [1]: IOError is OSError
Out[1]: True
```

so catching a `IOError` _should_ be totally equivalent to catching a `OSError`

Reply:
Ah ok I had not realised. Should be fine either way then.